lib/package.rb

```diff
@@ -48,9 +51,9 @@ def self.load_package(pkg_file)
   end
 
   def self.dependencies
-    # We need instance variable in derived class, so not define it here,
-    # base class.  Instead of define it, we initialize it in a function
-    # called from derived classees.
+    # We need instance variable in derived class, so do not define it here,
+    # base class.  Instead of defining it, we initialize it in a function
+    # called from derived classes.
```

Comment:
Kudos on attention to detail with the grammar and spell check.

Reply:
The privilege of growing up immersed in it...